shell/platform/android/android_surface.cc

```diff
@@ -0,0 +1,7 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/android/android_surface.h"
+
+namespace shell {}  // namespace shell
```

Comment:
By habit, there is always a source file for each header. If only to check that the header can be imported cleanly on its own (I know its not a fool-proof check for this). My editor created the stub for me and I didn't bother editing it after. Will get rid of the same.